Sprint-1/stretch/aoc-2018-day1/solution.js

```diff
@@ -0,0 +1,22 @@
+const fs = require("fs"); // Import the File System module
+
+// Read the file content
+const input = fs.readFileSync("input.txt", "utf8"); // 'utf8' ensures the data is read as a string
```

Comment:
"'utf8' ensures the data is read as a string" is not the reason why one specifies "utf8".